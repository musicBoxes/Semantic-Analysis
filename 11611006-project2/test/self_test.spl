int aha(){
	int a, b, c, d = 10, e[10];
}