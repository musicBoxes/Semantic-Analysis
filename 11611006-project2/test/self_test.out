Variable List
a int
b int
c int
d int
Function List
aha int
